src/main/java/zowe/client/sdk/zosjobs/input/GetJobParams.java

```diff
@@ -112,6 +112,10 @@ public static class Builder {
         private Integer maxJobs = JobsConstants.DEFAULT_MAX_JOBS;
         private String jobId;
 
+        public Builder() {
+            this.owner = "*";
+        }
+
```

Comment:
leave empty constructor body empty and do this instead:
private String owner = "*";